core/docker/build.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF 1>&2
+Usage: $0 [-v <VERSION>] [-h]
+Builds the Trino Docker image
+
+-h       Display help
+-v       Build the specified Trino version, downloads all required artifacts
+EOF
+}
+
+TRINO_VERSION=
+
+while getopts ":h:v:" o; do
+    case "${o}" in
+        v)
+            TRINO_VERSION=${OPTARG}
+            ;;
+        h)
+            usage
+            exit 0
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+shift $((OPTIND - 1))
+
+SOURCE_DIR="../.."
+WORK_DIR="$(mktemp -d)"
+
+# Retrieve the script directory.
+SCRIPT_DIR="${BASH_SOURCE%/*}"
+cd "${SCRIPT_DIR}" || exit 2
+
+if [ -n "$TRINO_VERSION" ]; then
+    SERVER_LOCATION="https://repo1.maven.org/maven2/io/trino/trino-server/${TRINO_VERSION}/trino-server-${TRINO_VERSION}.tar.gz"
+    CLIENT_LOCATION="https://repo1.maven.org/maven2/io/trino/trino-cli/${TRINO_VERSION}/trino-cli-${TRINO_VERSION}-executable.jar"
+    curl -o "${WORK_DIR}/trino-server-${TRINO_VERSION}.tar.gz" "${SERVER_LOCATION}"
+    curl -o "${WORK_DIR}/trino-cli-${TRINO_VERSION}-executable.jar" "${CLIENT_LOCATION}"
+    chmod +x "${WORK_DIR}/trino-cli-${TRINO_VERSION}-executable.jar"
+else
+    # Move to the root directory to run maven for current version.
```

Comment:
The "run maven" comment implies it is running a build, when it's actually just getting the current version

Reply:
I added the `-f ${SOURCE_DIR}` and now `mvnw` works without switching the dir, so I removed the comment.